TSPL.docc/GuidedTour/GuidedTour.md

```diff
@@ -2257,6 +2257,158 @@ print(fridgeIsOpen)
   ```
 -->
 
+## Concurrency
+
+You use `async` to mark an asynchronous function.
+You mark an asynchronous operation,
+like a call to an asynchronous function,
+by writing `await`
+if you want to wait for it to complete.
+
+```swift
+func bake(_ food: String) async -> String {
+    // ... wait for food to bake ...
+    return food
+}
+func makeCookies() async -> String {
+    let cookies = await bake("cookies")
+    return cookies
+}
+```
+
+<!--
+  - test: `guided-tour`
+
+  ```swifttest
+  -> func bake(_ food: String) async -> String {
+         // ... wait for food to bake ...
+         return food
+     }
+  -> func makeCookies() async -> String {
+         let cookies = await bake("cookies")
+         return cookies
+     }
+  ```
+-->
+
+You can use `async`-`let` to start an asynchronous operation
+without waiting for it to complete,
+which lets the operations run at the same time.
+Because you still need to wait for the operation to finish
+before you can use the value it returns,
+you write `await` when you read that constant's value.
+
+```swift
+async let cookies = makeCookies()
+async let bread = bake("bread")
+
+let bakedGoods = await [cookies, bread]
+```
+
+<!--
+  - test: guided-tour-async
+
+  ```swifttest
+  -> async let cookies = makeCookies()
+  -> async let bread = bake("bread")
+  ---
+  -> let bakedGoods = await [cookies, bread]
+  ```
+-->
+
+You structure concurrent code using task groups and child tasks.
+Writing `async`-`let` implicitly creates a child task.
+
```

Comment:
This example seems like it could you a touch more description. Maybe something about "creates a child task for each recipe and collects the baked desert at the end"?

Reply:
The pattern in the guided tour is to show the reader a sample of (most of) the language features in Swift and their syntax, without going into really any level of explanation.  None of the other code listings have text around them that explains what the code is doing.  So I'd rather leave this as is.